dspace-spring-rest/src/main/java/org/dspace/app/rest/model/ClaimedTaskRest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.dspace.app.rest.RestResourceController;
+
+/**
+ * The WorkflowItem REST Resource
+ *
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ */
+public class ClaimedTaskRest extends BaseObjectRest<Integer> {
```

Comment:
Please improve Javadocs. I don't think this is a "WorkflowItem"

Reply:
fixed https://github.com/DSpace/DSpace/pull/2312/files#diff-8afc90b30c652b5b004a16ef220c8124R15